celerytask/tasks.py

```diff
@@ -225,6 +225,26 @@ def update_all_discord_groups():
         update_discord_groups.delay(user.user_id)
 
 @task
+def update_discord_nickname(pk):
+    user = User.objects.get(pk=pk)
+    logger.debug("Updating discord nickname for user %s" % user)
+    authserviceinfo = AuthServicesInfo.objects.get(user=user)
+    character = EveManager.get_character_by_id(authserviceinfo.main_char_id)
+    logger.debug("Updating user %s discord nickname to %s" % (user, character.character_name))
+    try:
+        DiscordOAuthManager.update_nickname(authserviceinfo.discord_uid, character.character_name)
```

Comment:
Fair enough :) But beeing 1k + members, it's nice for fast corp identification of users. Thanks for the nickname addition :)